### DISCUSSION TITLE
File type search options

### DISCUSSION THREAD

**Discussion by ghost** [nicotine-plus/nicotine-plus#2108] [Q&A] [answered]
What possibilities are there besides:
```
flac|wav
!mp3
```
Can I do?
```
flac&wav
!flac&!wav
!(flac|wav)
```

**Comment by slook** [comment, marked as answer]
> What possibilities are there...

The only supported operators for the Filetype filter are:
`|` (or)
`!` (not)

See _Preferences > Searches > 'Result Filter Help':_

```
Multiple file extensions can be specified, which in turn will broaden the list of results.
    Example: flac|wav|ape

It is also possible to invert the filter, specifying file extensions you don't want in your results.
    Example: !mp3|!jpg
```

> `flac&wav`

A file cannot be both the `flac` _and_ `wav` types at the same time. Therefore the `&` operator is not supported.

> `!flac&!wav`
> `!(flac|wav)`

To exclude flac's and wav's from the results list the required syntax is `!flac|!wav`
